File: android/versioned-abis/expoview-abi32_0_0/src/main/java/abi32_0_0/expo/modules/camera/tasks/BarCodeScannerAsyncTaskDelegate.java
package abi32_0_0.expo.modules.camera.tasks;

import abi32_0_0.expo.interfaces.barcodescanner.BarCodeScannerResult;

public interface BarCodeScannerAsyncTaskDelegate {
  void onBarCodeScanned(BarCodeScannerResult barCode);
  void onBarCodeScanningTaskCompleted();
}

File: packages/expo-camera/android/src/main/java/expo/modules/camera/legacy/tasks/BarCodeScannerAsyncTaskDelegate.kt
package expo.modules.camera.legacy.tasks

import expo.modules.interfaces.barcodescanner.BarCodeScannerResult

interface BarCodeScannerAsyncTaskDelegate {
  fun onBarCodeScanned(barCode: BarCodeScannerResult)
  fun onBarCodeScanningTaskCompleted()
}
